workspaces/tests/batch_tx.rs

```diff
@@ -0,0 +1,41 @@
+use serde_json::json;
+use test_log::test;
+use workspaces::prelude::*;
+use workspaces::Function;
+
+#[test(tokio::test)]
+async fn test_batch_tx() -> anyhow::Result<()> {
+    let worker = workspaces::sandbox();
+    let contract = worker
+        .dev_deploy(include_bytes!("../../examples/res/status_message.wasm"))
+        .await?;
+
+    // Batch transaction with two `call`s into `set_status`. The second one
+    // should override the first one.
+    contract
+        .batch(&worker)
```

Comment:
perhaps `batch_transaction` might be more clear for a function name?

Reply:
I originally had `batch_tx` but thought it might've been redundant since we call `.transact()` at the end anyways which should signify that it's a transaction. And also was thinking that would be better for consistency since we don't have `call_transaction` either

Reply:
that's fair!